amazon/ec2credentials/client/src/main/kotlin/org/http4k/connect/amazon/ec2credentials/Ec2Credentials.kt

```diff
@@ -0,0 +1,27 @@
+package org.http4k.connect.amazon.ec2credentials
+
+import org.http4k.aws.AwsCredentials
+import org.http4k.connect.amazon.core.model.AccessKeyId
+import org.http4k.connect.amazon.core.model.Expiration
+import org.http4k.connect.amazon.core.model.SecretAccessKey
+import org.http4k.connect.amazon.core.model.SessionToken
+import java.time.Clock
+import java.time.Duration
+import java.time.ZonedDateTime
+
+data class Ec2Credentials(
```

Comment:
This was incorrectly placed in the amazon-core module; moving to the deprecated ec2credentials module